kyo-zio/shared/src/main/scala/kyo/ZIOs.scala

```diff
@@ -0,0 +1,53 @@
+package kyo
+
+import ZIOs.internal.*
+import kyo.kernel.*
+import scala.util.control.NonFatal
+import scala.util.control.NoStackTrace
+import zio.Cause
+import zio.Task
+import zio.ZIO
+
+opaque type ZIOs <: Async = GetZIO & Async
+
+object ZIOs:
+
+    def get[E >: Nothing: Tag, A](v: ZIO[Any, E, A])(using Frame): A < (Abort[E] & ZIOs) =
```

Comment:
Should params be by name throughout?

Reply:
I've added it to `run` but `get` doesn't seem possible because we need the concrete `ZIO` value to suspend the computation with